tests/arc-validate.Tests/CLITests/ValidateCommandTests.fs

```diff
@@ -22,7 +22,7 @@ let ``ValidateCommand CLI Tests`` =
                 testFixture (Fixtures.withToolExecution 
                     true
                     "../../../../../publish/arc-validate" 
-                    [|"-v"; "package"; "install"; "test"; "-pv"; "2.0.0"|]
+                    [|"-v"; "package"; "install"; "test"; "-pv"; "2.0.0"; "-pr"|]
```

Comment:
So here we install preview version 2.0.0, but below run `validate -p test -pv 2.0.0`. What if there is prod/preview 2.0.0 installed side-by side? we should only use preview packages here if there is a flag. 

I know that 2.0.0 should be equal on preview/prod but it is confusing nonetheless.